waterlog/src/ConfigRedux/middleware/signalRMiddleware.js

```diff
@@ -0,0 +1,57 @@
+import {
+    JsonHubProtocol,
+    HttpTransportType,
+    HubConnectionBuilder,
+    LogLevel
+} from '@aspnet/signalr';
+
+import {
+    FETCH_ALERTS_BEGIN
+} from '../../actions/Types';
+
+const onNotifReceived = res => {
+    console.log('****** NOTIFICATION ******', res);
```

Comment:
Do you need these logs

Reply:
File was no longer usable. Its been removed.